plugin/kubernetes/kubernetes.go

```diff
@@ -267,8 +268,7 @@ func (k *Kubernetes) InitKubeCache(ctx context.Context) (onStart func() error, o
 			k.APIConn.Run()
 		}()
 
-		timeout := 5 * time.Second
-		timeoutTicker := time.NewTicker(timeout)
+		timeoutTicker := time.NewTicker(k.startupTimeout)
```

Comment:
It is necessary to consider cases where settings are not configured to ensure backward compatibility.

Reply:
I have already set the default value of `startupTimeout` when executing the `ParseStanza` method in `setup.go` . The default value remains consistent with the current situation (5 seconds).